sim/rogue/item_sets_phase_8.go

```diff
@@ -218,9 +219,9 @@ func (rogue *Rogue) applyScarletEnclaveTank6PBonus() {
 		Label: label,
 		OnInit: func(aura *core.Aura, sim *core.Simulation) {
 			rogue.RollingWithThePunchesProcAura.ApplyOnStacksChange(func(aura *core.Aura, sim *core.Simulation, oldStacks int32, newStacks int32) {
-				if newStacks == 10 {
+				if newStacks == 5 {
 					energyAura.Activate(sim)
-				} else if newStacks < 10 && oldStacks == 10 {
+				} else if newStacks < 5 && oldStacks == 5 {
```

Comment:
I think we can just make this `else if oldStacks == 5` since if it's not `== 5` then it must be `< 5`

```suggestion
				} else if oldStacks == 5 {
```